cmd/internal/client/uninstall.go

```diff
@@ -1,11 +1,29 @@
 package client
 
 import (
+	"github.com/fuseml/fuseml/cli/kubernetes"
 	"github.com/fuseml/fuseml/cli/paas"
 	"github.com/pkg/errors"
 	"github.com/spf13/cobra"
 )
 
+var unInstallOptions = kubernetes.InstallationOptions{
```

Comment:
`unInstall` again, is it intended?

Reply:
This is for uninstall command, what's wrong?

Reply:
ok, you want to have `uninstallOptions`